nav.js

```diff
@@ -63,4 +63,83 @@ for (const impl_collapsed of document.getElementsByClassName('impl_collapsed'))
         impl_args[0].addEventListener('click', () =>
             impl_collapsed.classList.remove('impl_collapsed'));
     }
-}
\ No newline at end of file
+}
+
+
+
+function filterSelectionClass(c, classname) {
+    var x, i;
+    x = document.getElementsByClassName(classname);
+    //if (c == "all") c = "";
+    //cs = c.split(",");
+    // Add the "show" class (display:block) to the filtered elements, and remove the "show" class from the elements that are not selected
+    for (i = 0; i < x.length; i++) {
+      w3AddClass(x[i], "hide");
+      for (j = 0; j < c.length; j++) { 
+          if (x[i].className.indexOf(c[j]) > -1) w3RemoveClass(x[i], "hide");
+      }
+    }
+  }
+
+  function filterSelection(c) {
+    filterSelectionClass(c, "tactic");
+    filterSelectionClass(c, "taclink");
+  }
+
+// Show filtered elements
+function w3AddClass(element, name) {
```

Comment:
```suggestion
```

`element.classList.{add,remove}` is now widely supported and we should use it: https://caniuse.com/#feat=classlist

Reply:
Ah, thanks! My JS knowledge doesn't go beyond copying code from StackOverflow.